kogito-business-rules-springboot/README.md

```diff
@@ -12,6 +12,36 @@ This example shows
 	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
 
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties2.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramPropertiesDataImports.png"></p>
+
+* Evaluate Person Business Rule
+<p align="center"><img src="docs/images/evaluatePersonBusinessRule.png"></p>
+
+* Evaluate Person Business Rule
+<p align="center"><img src="docs/images/evaluatePersonBusinessRule2.png"></p>
+
+* Evaluate Person Business Rule
+<p align="center"><img src="docs/images/evaluatePersonBusinessRuleDataAssignments.png"></p>
+
+* Special Handling for Children
+<p align="center"><img src="docs/images/specialHandlingForChildren.png"></p>
+
+* Special Handling for Children
```

Comment:
`Special Handling for Children (middle)`

Reply:
fixed @radtriste